src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
The thread got long and a lot of it is unrelated to the fullscreen issue.
:) I'll write a summary for you when I get home (I'm writing from a mobile
right now).

-- 
Michał Gołębiowski